packages/suite/src/views/dashboard/components/AssetsCard/index.tsx

```diff
@@ -193,6 +193,7 @@ const AssetsCard = () => {
                                 color={theme.TYPE_RED}
                                 size={14}
                             />
+                            &nbsp;
```

Comment:
Please style the `Icon` instead. It should be styled with CSS instead of the `style` property which is not read anyway.

Reply:
😨 1599f9c398e028a2e171dfa8716a5e24abfac528